### ISSUE TITLE
Build failed: please ensure the correct public key is added to the list of trusted keys for this repository and the remote branch exists.

### ISSUE THREAD

**Issue by sureshchouksey** [kevana/ui-for-docker#210]
Hi,
     I have forked this project to my github account and integrate with dockerhub. I want to commit anything in github or any changes online in github then build on dockerhub ,so there are some problem generate in docker.

"Build failed: please ensure the correct public key is added to the list of trusted keys for this repository and the remote branch exists."

How to solve this problem.

**Comment by davivcgarcia** [comment]
This sounds like a problem with your SSH keys, not with the repository itself. As a test, I just forked the project and managed to create the Automated Build without problems. You should be following the instructions from [here](https://github.com/kevana/ui-for-docker/wiki/How-to-Build-a-release).

**TL;DR:** _You should use the branch "dist" instead of "master" to create the Docker Hub Automated Build._

**Comment by kevana** [comment]
@davivcgarcia Yep, sounds similar to [this StackOverflow question](http://stackoverflow.com/questions/35598365/automated-build-from-github-private-repositority-on-docker-hub-doesnt-work/35634742), unrelated to the ui-for-docker code.
